gilbreth_perception/src/segmentation_node.cpp

```diff
@@ -12,18 +12,27 @@
 
 // Algorithm params
 float down_sample(0.01);
+double x_l, x_u, y_l, y_u, z_l, z_u;
 
 ros::Publisher pub;
 
 void loadParameter() {
   // General parameters
   std::map<std::string, float> parameter_map;
+  XmlRpc::XmlRpcValue camera_scope;
   ros::NodeHandle ph("~");
   ph.getParam("parameters", parameter_map);
   down_sample = parameter_map["down_sample"];
+  ph.getParam("camera_scope", camera_scope);
+  x_l=camera_scope["x"][0];
+  x_u=camera_scope["x"][1];
+  y_l=camera_scope["y"][0];
+  y_u=camera_scope["y"][1];
+  z_l=camera_scope["z"][0];
+  z_u=camera_scope["z"][1];
```

Comment:
are these used to select a ROI from the point cloud?

Reply:
Yes, it is.

Reply:
I think a better name would be "roi" instead of camera scope.

Reply:
OK, I will change it, it does sound better.